packages/design-system/src/Modal/Modal.styles.tsx

```diff
@@ -0,0 +1,80 @@
+import styled, { keyframes } from "styled-components";
+import { Overlay, Content, Close } from "@radix-ui/react-dialog";
+import {
+  ModalContentBodyClassName,
+  ModalContentFooterClassName,
+} from "./Modal.constants";
+
+const overlayShow = keyframes`
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+const contentShow = keyframes`
+  from {
+    opacity: 0;
+    transform: translate(-50%, -50%) scale(0.9);
+  }
+  to {
+    opacity: 1;
+    transform: translate(-50%, -50%) scale(1);
+  }
+`;
+
+export const StyledOverlay = styled(Overlay)`
+  /* TODO: Opacity for colors needs to be handled in a better way*/
+  background-color: #39393999;
```

Comment:
Could you comment the original theme variable this is from?

Reply:
This color is not part of new design system.